cmd/zrepl/zrepl.c

```diff
@@ -211,13 +218,17 @@ uzfs_zvol_io_receiver(void *arg)
 			}
 		}
 
+		if (zinfo->state == ZVOL_INFO_STATE_OFFLINE) {
+			zio_cmd_free(&zio_cmd);
+			goto exit;
```

Comment:
IMHO the fewer goto statements and labels the better code and here we don't really need it. "break" is sufficient.

Reply:
I agree, goto statements are not desirable. This code is by-product of conflict resolution. Done.